### ISSUE TITLE
The class Routes is not defined anymore

### ISSUE THREAD

**Issue by naamapps** [Milad-Akarie/auto_route_library#389]
After upgrading, everywhere in my project that I use `Routes.*route_name*` is now obsolete because the autoroute generator doesn't generate the `Routes` class anymore.

Is it intentional?
If so, why did you make this decision and what is the alternative?
Thanks.

**Comment by theweiweiway** [comment]
Hey @naamapps, do you mean upgrading from AutoRoute 0.69 to 1.0?

**Comment by naamapps** [comment]
Hi @theweiweiway, Yes exactly.
Upgraded AutoRoute and AutoRouteGenerator to 1.0

**Comment by theweiweiway** [comment]
Please see this page to see how 1.0.x creates Routes:

https://autoroute.vercel.app/basics/root_router

Specifically, check out the `replaceInRouteName` field. This replaces the `Routes.<ROUTE_NAME>` way of how 0.69 generated routes

**Comment by AlexMeuer** [comment]
**tl;dr** Access routes directly now. Replace `Routes.myRoute` with `MyRoute()`.
